loadflow/loadflow-api/src/test/java/com/powsybl/loadflow/LoadFlowDefaultParametersLoaderTest.java

```diff
@@ -128,4 +131,62 @@ void testProviderParameters() {
         assertEquals(5, afterPlatformConfig.getParameterDouble());
         assertEquals("modified", afterPlatformConfig.getParameterString());
     }
+
+    private boolean sameMetaData(List<Parameter> l1, List<Parameter> l2) {
+        // Everything except default value shoudl be the same
+        assertEquals(l1.size(), l2.size());
+        Iterator<Parameter> it1 = l1.iterator();
+        Iterator<Parameter> it2 = l2.iterator();
+        for (int i = 0; i < l1.size(); i++) {
+            Parameter p1 = it1.next();
+            Parameter p2 = it2.next();
+            assertEquals(p1.getName(), p2.getName());
+            assertEquals(p1.getCategoryKey(), p2.getCategoryKey());
+            assertEquals(p1.getDescription(), p2.getDescription());
+            assertEquals(p1.getPossibleValues(), p2.getPossibleValues());
+            assertEquals(p1.getType(), p2.getType());
+            assertEquals(p1.getScope(), p2.getScope());
+        }
+        return true;
+    }
+
+    @Test
+    void testSpecificParameters() {
+        LoadFlowDefaultParametersLoaderMock loader = new LoadFlowDefaultParametersLoaderMock("test");
+        LoadFlowProvider provider = new LoadFlowProviderMock();
+        List<Parameter> parameters = provider.getSpecificParameters();
+        List<Parameter> parametersWithDefaultValue = LoadFlowProviderUtil.getSpecificParameters(provider, Optional.of(loader));
+
+        assertTrue(sameMetaData(parameters, parametersWithDefaultValue));
```

Comment:
To avoid this unnecessary `assertTrue`, you can rename `sameMetaData` into `assertSameMetaData`.